public/give.html

```diff
@@ -305,54 +305,71 @@
          * END MAP SEARCH STYLES *
          *************************/
 
+         /* Overall page styles */
+         .translated-content {
+           display: none;
+         }
     </style>
 
 
 </head>
 
 <body>
 <main class="content">
-    <div class="container" style="padding-bottom: 40px;">
-      <div class="map-container" style="display: none;">
+    <div class="languages-loading" style="position: fixed;top: 50%; left: 50%;">
+        <div class="spinner-border" role="status">
+          <span class="sr-only">Loading translations...</span>
```

Comment:
should this be translated as well?

Reply:
Not this one. It appears before translations are loaded, so it can't be. I am hoping that the number of non-English-speaking blind users we have will be small enough that it'll be ok. :)